src/assets/scripts/templates/index.js

```diff
@@ -0,0 +1,10 @@
+import '../sections/product';
+
+import $ from 'jquery';
+import sections from '@shopify/theme-sections';
+
+$(document).ready(() => {
+  sections.load([
```

Comment:
For a single section, you could do: `sections.load('product');` - but I guess this is more future proof in case we add any additional sections to load.

Reply:
yes :)